src/adapters/parcel-rights-fetcher.ts

```diff
@@ -45,19 +60,42 @@ export type ParcelOperatorsFromQuery = {
       owner: {
         address: string
       }
+      operator: string | null
       updateOperator: string | null
     }
   ]
 }
 
 export type ParcelOperators = {
   owner: string
-  operator?: string
+  operators: string[]
```

Comment:
Do we need an array for `operator` and `updateOperator`? for each asset you can have just one

Reply:
Changed it to only having one operator and updateOperator.